polygon/websocket/models/common.py

```diff
@@ -9,6 +9,16 @@ class Feed(Enum):
     PolyFeedPlus = "polyfeedplus.polygon.io"
     StarterFeed = "starterfeed.polygon.io"
     Launchpad = "launchpad.polygon.io"
+    Business = "business.polygon.io"
+    EdgxBusiness = "edgx-business.polygon.io"
+    DelayedBusiness = "delayed-business.polygon.io"
+    DelayedEdgxBusiness = "delayed-edgx-business.polygon.io"
+    DelayedNasdaqLastSaleBusiness = "delayed-nasdaq-last-sale-business.polygon.io"
+    DelayedNasdaqBasic = "delayed-nasdaq-basic-business.polygon.io"
+    DelayedFullMarketBusiness = "delayed-fullmarket-business.polygon.io"
+    FullMarketBusiness = "fullmarket-business.polygon.io"
+    NasdaqfeedLastSaleBusiness = "nasdaq-last-sale-business.polygon.io"
+    NasdaqfeedBasicBusiness = "nasdaq-basic-business.polygon.io"
```

Comment:
should we rename these variables to be `NasdaqLastSaleBusiness` and `NasdaqBasicBusiness` for consistency with the endpoints?

Reply:
done!